docs/user/spark/jts_spark.rst

```diff
@@ -0,0 +1,103 @@
+JTS Spark
+---------
+
+The JTS Spark module provides a set of User Defined Functions (UDFs) and User
+Defined Types (UDTs) that enable executing SQL queries in spark that perform
+geospatial operations on geospatial data types.
+on
+GeoMesa SparkSQL support builds upon the ``DataSet``/``DataFrame`` API present
+in the Spark SQL module to provide geospatial capabilities. This includes
+custom geospatial data types and functions, the ability to create a ``DataFrame``
+from a GeoTools ``DataStore``, and optimizations to improve SQL query performance.
+
+This functionality is located in the ``geomesa-jts-spark`` module:
+
+.. code-block:: xml
+
+    <dependency>
+      <groupId>org.locationtech.geomesa</groupId>
+      <artifactId>geomesa-jts-spark_2.11</artifactId>
+      // version, etc.
+    </dependency>
+
+Example
+^^^^^^^
+
+The following is a Scala example of loading a DataFrame with user defined types
+
+.. code-block:: scala
+
+    val schema = StructType(Array(StructField("name",StringType, nullable=false),
+                                StructField("polygonText", StringType, nullable=false),
+                                StructField("latitude", DoubleType, nullable=false),
+                                StructField("longitude", DoubleType, nullable=false)))
+
+    val dataFile = this.getClass.getClassLoader.getResource("jts-example.csv").getPath
+    df = spark.read.schema(schema).option("sep", "-").csv(dataFile)
+
+    val polyFromText = udf(SQLGeometricConstructorFunctions.ST_PolygonFromText)
+    val pointFromCoord = udf(SQLGeometricConstructorFunctions.ST_MakePoint)
+    alteredDF = df.withColumn("polygon", polyFromText(col("polygonText")))
+                  .withColumn("point", pointFromCoord(col("latitude"), col("longitude")))
+
+
+
+Notice how the initial schema does not have a UserDefinedType, but after applying our
+User Defined Functions to the appropriate columns, we are left with a data frame with
+geospatial column types.
+
+Configuration
+^^^^^^^^^^^^^
+
+To enable this behavior, you must pass the  `SQLContext` of the `SparkSession` to
```

Comment:
i think you want double tick marks here, for code snippets? currently it's rendered as italics.